src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -312,6 +331,16 @@ protected override Encoding StandardOutputEncoding
 
         protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
```

Comment:
The one additional thing the Vbc task does in Vbc.Execute after calling base.Execute is to rename the .pdb generated by the compiler since the compiler currently does not allow specifying the .pdb name through command-line arguments. Vbc.Execute will attempt to rename the .pdb if base.Execute returns true.